docs/content/querying/sql.md

```diff
@@ -519,6 +524,101 @@ SELECT * FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_SCHEMA = 'druid' AND TABLE_
 |COLLATION_NAME||
 |JDBC_TYPE|Type code from java.sql.Types (Druid extension)|
 
+## SYSTEM SCHEMA
+
+The sys schema provides visibility into Druid segments, servers and tasks.
```

Comment:
I think it's good to put it in quotes, so like: The "sys" schema.

Reply:
Actually, I am a bit confused, and not sure if `sys` would sound better at all places. @gianm do you have an opinion? Should we make is consistent and call it `sys` instead of `system` at all places.

Reply:
added the quotes around sys

Reply:
Quotes look good. It makes clear that `sys` is the real name.